src/autowire.js

```diff
@@ -19,6 +19,50 @@ function registerRoutes (Vue, requireContext) {
   });
 }
 
+/**
+ * Load router files
+ * @param {Vue} Vue VueJS instance
+ * @param {Object} requireContext Webpack's require context. See https://github.com/webpack/docs/wiki/context#context-module-api
+ */
+function registerFilters (Vue, requireContext) {
+  // Ask webpack to list the files
+  // By default require.context adds all files to the main bundle unless "lazy" mode is used
+  const filterFiles = requireContext.keys();
+
+  return filterFiles.map(file => {
+    const name = getComponentName(file);
```

Comment:
Might need to rename `getComponentName` into something else. `getAssetName` maybe?

Reply:
Done.